lib/membrane_g711/lut_builder.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Membrane.G711.LUTBuilder do
+  @moduledoc false
+
+  import Bitwise
+
+  @alaw_mask_1 0x55
+  @alaw_mask_2 0xD5
```

Comment:
This naming is a little confusing for me as there is only one mask in the specification, and it refers to the same value as our constant `alaw_mask_2`. Maybe the first should be named `g711_mask`?